migrations/current.ts

```diff
@@ -1,5 +1,186 @@
-// write plv8 based migration here
```

Comment:
My apologies for the in-progress nature of the README on this stuff. :sweat_smile: 

This file is where you would work on your migration locally but once you `graphile-migrate commit`, we should copy it to a file in the migrations/src directory with a name corresponding to it's respective .sql file in migrations/committed and then current.ts.